### PR TITLE
Fix #61 - Migrate to new runtime client, take II

### LINKED ISSUE #61: Puppet 7 gives deprecation warning
```
Warning: The method 'Puppet::Network::HttpPool.http_instance' is deprecated. Use Puppet.runtime[:http] instead
   (file & line not available)
```
See here:

https://github.com/voxpupuli/puppet-healthcheck/blob/master/lib/puppet_x/puppet-community/http_validator.rb#L38

### PR DESCRIPTION
**Pull request by kbucheli**
#### Pull Request (PR) description
Based on PR #62 I changed a few accessors which are different with the new runtime.

#### This Pull Request (PR) fixes the following issues
Fixes #61

### CODE REVIEW

File: lib/puppet_x/puppet-community/http_validator.rb [hunk 1]
```diff
@@ -35,16 +33,16 @@ def initialize(http_resource_name, http_server, http_port, use_ssl, test_path, e
       #
       # @return true if the connection is successful, false otherwise.
       def attempt_connection
-        conn = Puppet::Network::HttpPool.http_instance(http_server, http_port, use_ssl, verify_peer)
+        conn = Puppet.runtime[:http]
```
[the remark this answers is not in the document]
Comment (kbucheli) [reply lib/puppet_x/puppet-community/http_validator.rb:36]: Thanks for the input, done.

**Review by bastelfreak — APPROVED** [review]

**Comment by joshcooper** [comment]
LGTM though I don't have a way to test

**Comment by kbucheli** [comment]
Is there anything missing for merging?

**Comment by jcpunk** [comment]
Any objection to getting this merged?

**Pull request merged**